src/electron/lib/api/apis/UtilApi.ts

```diff
@@ -68,6 +72,20 @@ export class UtilApi implements ElectronMainApi<UtilApi> {
           },
         ],
       },
+      {
+        id: "keybind_settings",
+        title: "Keybindings",
+        settings: [
+          {
+            id: "Keybindings",
+            title: "Keybindings",
+            subtitle: "Customize your keybindings",
+            type: "preferences",
+            secret: false,
+            value: getPreferences(),
```

Comment:
In context of the comment in the `saveUserSettings` method, the keybindings will/should be stored with the key of `Keybindings`. This key needs to be used to retrieve the settings from `electron-store` directly using `settings.get("Keybindings")`.